### PR TITLE
fix scroll to module

### PR DESCRIPTION
**Pull request by balintbartok**
refs: MBL-14961
affects: student, teacher
release note: Fixed module scrolling in special cases

test plan: see ticket, make sure that cache is cleared

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/56201768/p/61faad3f3c5c4dc73d5d1dd0c927cc46">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/56201778/p/00ca768fea6fca5032e27637f55f4326">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/56201725/p/71f69efef1de11141e27f2a19b6b9d49">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  One or more files are below the minimum test coverage 50%
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 92.04% | 92.04% | 0%
Core/Core/Dashboard/DashboardInviteSection.swift | 0% | 0% | 0%
Core/Core/Submissions/GradeSubmission.swift | 0% | 0% | 0%
Core/Core/Dashboard/DashboardCourseSection.swift | 36.67% | 36.67% | 0%
Core/Core/Dashboard/DashboardConferenceSection.swift | 0% | 0% | 0%
Core/Core/Extensions/TransactionExtensions.swift | 0% | 0% | 0%
Core/Core/Dashboard/DashboardAnnouncementSection.swift | 0% | 0% | 0%

[MBL-14961](https://instructure.atlassian.net/browse/MBL-14961)

#### Affected Apps: Student, Teacher

#### Release Note: 
Fixed module scrolling in special cases
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Modules/ModuleList/ModuleListViewController.swift [hunk 1]
```diff
@@ -145,8 +145,8 @@ public class ModuleListViewController: UIViewController, ColoredNavViewProtocol,
 
     func scrollToModule() {
         if let moduleID = moduleID, let section = modules.all.firstIndex(where: { $0.id == moduleID }), section < tableView.numberOfSections {
-            let rect = tableView.rect(forSection: section)
-            tableView.setContentOffset(CGPoint(x: 0, y: rect.minY), animated: true)
+            let indexPath = IndexPath(row: 0, section: section)
+            tableView.scrollToRow(at: indexPath, at: .top, animated: true)
```
Comment (vanwagonet) [inline Core/Core/Modules/ModuleList/ModuleListViewController.swift:149]: nice, this is a much more reliable call.

**Review by vanwagonet — COMMENTED** [review]
Code change looks great.

**Review by vanwagonet — APPROVED** [review]
QA+1

**Pull request merged**